### ISSUE TITLE
readtags command

### ISSUE THREAD

**Issue by DrPyser** [universal-ctags/ctags-snap#29]
Hi!

It seems the snap package does not include the `readtags` command shipped with `ctags` in other packaging. I cannot seem to find a separately packaged `readtags` either. 
So I either have to build `ctags` from source or install a four years old version of ctags from the apt repository...

Is there an alternative? Could `readtags` be packaged with this snap, or in a separate snap?

Thanks for this great tool!

**Comment by tartley** [comment]
Good thought. I'll check it out...

**Comment by tartley** [comment]
I currently have no idea how to include a 2nd executable in a snap. It might be easy. If anyone reading has any hints, they are welcome.

**Comment by DrPyser** [comment]
https://forum.snapcraft.io/t/adding-a-second-command-executable/12837

**Comment by tartley** [comment]
Thank you!

**Comment by tartley** [comment]
When solving this, we could presumably also tackle #8

**Comment by masatake** [comment]
https://github.com/gediminasz/ctags-companion/issues/73#issue-3941747500

**Comment by tartley** [comment]
Thanks for the link/notification. For transparency: I'm still a happy ctags user but no longer have the motivation for maintaining the snap distribution I once did (I have left the Canonical Snap Store team), so I won't be implementing this myself. I'll stay subscribed to all these issues and will happily assist and/or handover to anyone who wants to.

**Comment by masatake** [comment]
> For transparency: I'm still a happy ctags user ...

Do you use ctags via snap?

**Comment by tartley** [comment]
> Do you use ctags via snap?

No. I no longer use snaps for anything on my laptop/desktop, where I think that flatpak has won the battle, but I do still use snaps for some server builds, where I believe they are a better fit (e.g. slow startup time is of less concern for long-running services).

**Comment by masatake** [comment]
Don't you change the status of this project `archive`?

**Comment by tartley** [comment]
I have not changed the project status to archive. I do not currently intend to. The project continues to automatically build snaps containing the latest ctags code. I plan to just leave it that way, and if anyone else wants to address issues or feature requests then they can.

Let me know if you'd prefer me to do something different.
